pwiz_tools/Skyline/TestUtil/TestFunctional.cs

```diff
@@ -334,6 +334,27 @@ public static void RunUI([InstantHandle] Action act)
             });
         }
 
+        /// <summary>
+        /// Convenience function for getting a value on the UI thread
+        /// e.g. var value = GetValueUI(() => control.Value);
+        /// </summary>
+        public T GetValueUI<T>([InstantHandle] Func<T> act)
```

Comment:
I would recommend naming this function "CallUI".
It does the same thing as "RunUI", except that it takes a Func instead of an Action.

You might as well make use of "RunUI" in the implementation:
```
        public static T CallUI<T>([InstantHandle] Func<T> func)
        {
            T result = default;
            RunUI(()=>result = func());
            return result;
        }
```

Reply:
I'll try that, thanks.

Reply:
Yes, much tidier, thanks.